dbAlchemy.py

```diff
@@ -0,0 +1,43 @@
+from itertools import chain
+
+import sqlalchemy as db
+
+
+
+class DbSqlAlchemy:
+    def __init__(self):
+        self.engine = db.create_engine('sqlite:///./Database/info.db', connect_args={'check_same_thread': False},
```

Comment:
hardcoded DB config. `sqlite:///./Database/info.db`
This information must be passed via the constructor argument.

Reply:
Done